app/controllers/orders_controller.rb

```diff
@@ -25,18 +28,93 @@ def empty_cart
   end
 
   def checkout
+    if @cart.check_branch_inventory?
+      @cart.update(order_params)
+      @cart.update_inventory(false)
+      stripe_session = create_stripe_session
+      @payment = @cart.payments.create(user: @current_user, stripe_session_id: stripe_session.id)
```

Comment:
We should save user's stripe key in user so that for next payment we use the same key

Reply:
Done